.github/workflows/build_binaries.yaml

```diff
@@ -0,0 +1,42 @@
+name: Build and publish binaries
+on:
+  release:
+    types:
+      - created
+jobs:
+  build:
+    name: Build and publish binaries
+    strategy:
+      matrix:
+        go-version: [ 1.16.x ]
+        platform: [ ubuntu-latest ]
+    runs-on: ${{ matrix.platform }}
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v1
+      with:
+        go-version: ${{ matrix.go-version }}
+    - name: Check out code
+      uses: actions/checkout@v2
+    - name: Build Binary for linux amd64
+      run: go build -o lndhub-${{github.event.release.tag_name}}-linux-x86_64
+    - name: Build Binary for linux arm v7
+      run: GOOS=linux GOARCH=arm GOARM=7 go build -o lndhub-${{github.event.release.tag_name}}-linux-arm_v7
+    - name: Upload amd64 binary to release assets
+      uses: actions/upload-release-asset@v1.0.1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        upload_url: ${{ github.event.release.upload_url }}
+        asset_path: ./lndhub-${{github.event.release.tag_name}}-linux-x86_64
+        asset_name: lndhub-${{github.event.release.tag_name}}-linux-x86_64
+        asset_content_type: binary/octet-stream
+    - name: Upload armv7 binary to release assets
+      uses: actions/upload-release-asset@v1.0.1
```

Comment:
this seems deprecated: https://github.com/actions/upload-release-asset
they refer to this one as maintained: https://github.com/softprops/action-gh-release

Reply:
If it works, I don't see why we couldn't use it.

Reply:
The one you linked to is for creating a release with assets, not uploading assets to an existing one.

Reply:
I just normally like to avoid unmaintained code. 
but works! 👍